main.py

```diff
@@ -232,9 +312,10 @@ def escape_path(path):
 		accelerator_arg = Xqemu.generateAcceleratorArg(settings.settings['use_accelerator'])
 
 		dvd_path_arg = ''
-		if settings.settings['dvd_present']:
-			check_path(settings.settings['dvd_path'])
-			dvd_path_arg = ',file=' + escape_path(settings.settings['dvd_path'])
+		if main.tableGames.currentRow() > 0:
```

Comment:
I assume this case handles if nothing is selected? What's the return value then?
I couldn't find good information in the Qt docs - what's your source for this check?

Reply:
Its `-1` if nothing is selected but 0 wont help me either cause the first entry in the library list is ALWAYS the `No disc in tray` option so one can start the dashboard.